resources/logo.svg

```diff
@@ -1,46 +1,44 @@
-<?xml version="1.0" encoding="utf-8"?>
+<?xml version="1.0" encoding="utf-8" standalone="yes"?>
```

Comment:
I would suggest keeping the logos in separate files. This leaves them more easily maintainable, especially given that one is from the design team who also has a copy.